wwwroot/lib/jquery-ajax-unobtrusive/jquery.unobtrusive-ajax.js

```diff
@@ -0,0 +1,205 @@
+// Unobtrusive Ajax support library for jQuery
```

Comment:
Everything under wwwroot/lib should be in gitignore and not committed, it's already in libman.json so these files here can just be removed